multisite/models.py

```diff
@@ -142,7 +142,7 @@ class NotCanonicalAliasManager(models.Manager):
 
     def get_queryset(self):
         qset = super(NotCanonicalAliasManager, self).get_queryset()
-        return qset.filter(is_canonical__isnull=True)
+        return qset.filter(is_canonical=False)
```

Comment:
@kutera if we don't set a default on the field class then we don't need this change

Reply:
This change avoid deprecated warning and error in future django release.
In django 3.1 See https://docs.djangoproject.com/en/dev/releases/3.1/#deprecated-features-3-1
"The undocumented usage of the isnull lookup with non-boolean values as the right-hand side will no longer be allowed."

Reply:
@kutera, I had missed that, thanks for pointing this out.